src/gridstack.ts

```diff
@@ -5,7 +5,7 @@
  * Copyright (c) 2021 Alain Dumesny
  * see root license https://github.com/gridstack/gridstack.js/tree/master/LICENSE
  */
-
+require('@babel/polyfill')

```

Comment:
no, I don't want modern browsers to compile poyfill. Only the few IE users can include our polyfill js file instead...